ivy/functional/frontends/torch/Tensor.py

```diff
@@ -126,7 +126,12 @@ def _to_with_device(
         return ivy.asarray(self.data, device=device, dtype=dtype, copy=copy)
 
     def arctan(self, *, out=None):
-        return torch_frontend.arctan(self, out=out)
+        return torch_frontend.atan(self, out=out)
+
+    @with_unsupported_dtypes({"1.11.0 and below": ("bfloat16")}, "torch")
+    def arctan_(self):
```

Comment:
You should import the decorator to use it, that's why the lint check is failing.
`from ivy.func_wrapper import with_unsupported_dtypes`

Reply:
By the way, if you click on Details next to the failing check, you can see the error messages 🙂